wordpress-objects/Clarkson_Post_Type.php

```diff
@@ -34,24 +34,36 @@ public static function get( string $post_type ): Clarkson_Post_Type {
 	}
 
 	/**
-	 * Check if an existing field is requested.
+	 * Proxy requested properties to WP Post Type if it doesn't exist in Clarkson Post Type
 	 *
 	 * @param string $name Field to search by.
-	 *
 	 * @throws Exception Error message.
 	 */
 	public function __get( string $name ) {
-		if ( in_array( $name, array(), true ) ) {
-			throw new Exception( 'Trying to access wp_post_type object properties from Post Type object.' );
+		if ( property_exists( $this, $name ) ) {
+			return $this->name;
```

Comment:
Nice work, does this also returns private properties? Because of so then, a check for this might be a good thing. 

https://www.php.net/manual/en/reflectionclass.getproperties.php#refsect1-reflectionclass.getproperties-examples

Reply:
reflectionclass was not required, as __get and __isset are only called when trying to access a protedcted, private or unexisting property. So we can ignore retreiving the property from the current object, and try retreiving it from the WP_POST_TYPE object. That should fail if it's private or protected.